src/discordcr/client.cr

```diff
@@ -764,6 +766,9 @@ module Discord
       end
     end
 
+    # :nodoc:
```

Comment:
Should probably add some documentation to the method so that it appears within our docs.
Just something that mentions that the event is called whenever the websocket receives any sort of frame/message.